src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -1,21 +1,39 @@
 ARG SDK_BASE_IMAGE=mcr.microsoft.com/dotnet/nightly/sdk:6.0-bullseye-slim
 FROM $SDK_BASE_IMAGE
 
-WORKDIR /app
-COPY . .
 
 # Pulling the msquic Debian package from packages.microsoft.com
+# Build latest msquic locally
+WORKDIR /msquic
 RUN apt-get update -y
-RUN apt-get install -y gnupg2 software-properties-common
-RUN curl -sSL https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
-RUN apt-add-repository https://packages.microsoft.com/debian/11/prod
-RUN apt-get update -y
-RUN apt-get install -y libmsquic
 RUN apt-get upgrade -y
+RUN apt-get install -y cmake clang ruby-dev gem lttng-tools libssl-dev && \
+    gem install fpm
+RUN git clone --recursive https://github.com/dotnet/msquic
+RUN cd msquic/src/msquic && \
+    mkdir build && \
+    options="-DQUIC_BUILD_TOOLS=off -DQUIC_BUILD_TEST=off -DQUIC_BUILD_PERF=off" && \
```

Comment:
I'm just curious, is this line needed? the options are specified below in the call once again...

Reply:
I copied the code from Manicka's branch :D but agree that it looks unnecessary

Reply:
No they're not. This is a remnant of a salvaged code from my personal script for building msquic :smile: